.azure-pipelines/azure_devops.yml

```diff
@@ -1,22 +1,47 @@
-name: "$(Year:yyyy).$(Month).$(BuildID)-ci"
-
-trigger:
-  batch: true
-  branches:
-    include:
-      - main
-
-pr: none
-
-resources:
-  repositories:
-    - repository: openjdk_azure_devops_harness
-      type: git
-      name: github_move/openjdk-azure-devops-harness
-      ref: refs/heads/main
-
-variables:
-  JTREG_VERSION: "6.1"
-
-stages:
-  - template: /ci_main.yml@devops_demo
\ No newline at end of file
+name: "$(Year:yyyy).$(Month).$(BuildID)"
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+
+pr: none
+
+resources:
+  repositories:
+    - repository: jdk-harness
+      type: git
+      name: Devdiv/jdk-harness

```

Comment:
Q: Still require a `name:` field when specifying a `repository:` and `project:` ? 

(Could be a best practice to include anyways?)

Reply:
All the fields would reference to Devdiv jdk-harness. Originally it worked without having project: but was throwing errors when I was testing it out.